lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -167,13 +176,48 @@
                 }
             ];
 
+            ordersHistoryData.unshift(
+                {
+                    'title': gettext('ORDER NAME'),
+                    'order_date': gettext('ORDER PLACED'),
+                    'price': gettext('TOTAL'),
+                    'number': gettext('ORDER NUMBER')
+                }
+            );
+
+            ordersSectionData = [
+                {
+                    title: gettext('My Orders'),
+                    subtitle: window.interpolate_text(
+                        gettext('This page contains information about orders that you have placed with {platform_name}.'), {platform_name: platformName}    /* jshint ignore:line */
```

Comment:
@AlasdairSwan I moved interpolate parameters to next line but line is still longer then the limit and we can't break text string into two line because @andy-armstrong mention that ```gettext``` will not work on it if we break it into multiple lines.